### ISSUE TITLE
SAF support

### ISSUE THREAD

**Issue by goddaneel** [Acclorite/book-story#38]
Would like to be able to support saf (Storage Access Framework ) if possible, rather than just system-wide reads.

**Comment by Acclorite** [comment]
No. The reasons you can read [here](https://github.com/Acclorite/book-story/issues/36). Also, the app is reproducible(e.g. [IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/ua.acclorite.book_story)'s green checkmark), meaning there is nothing added to the APK, if we talk about trust.

**Comment by goddaneel** [comment]
> No. The reasons you can read [here](https://github.com/Acclorite/book-story/issues/36). Also, the app is reproducible(e.g. [IzzyOnDroid](https://apt.izzysoft.de/fdroid/index/apk/ua.acclorite.book_story)'s green checkmark), meaning there is nothing added to the APK, if we talk about trust.

I think you probably misunderstood me, it doesn't stem from a trust issue, it's simply that I personally prefer a saf based management framework.
I actually don't mind the global read access, I have a lot of software on my devices that have been given that access.
What I'm asking about is the difficulty of implementing this, if it would make development more difficult or you don't feel it's necessary, then it's not a necessary feature.

**Comment by Acclorite** [comment]
Yeah, I am sure it would make the development more difficult, at least now. I don't mind changing ways of getting books, just for now I want to focus on my 30+ TODOs and 15 opened issues.. If I will reconsider implementing it, I will open this issue again.

**Comment by goddaneel** [comment]
> Yeah, I am sure it would make the development more difficult, at least for now. I don't mind changing ways of getting books, but at this moment I want to focus on my 30+ TODOs and 15 opened issues.. If I will reconsider implementing it, I will open this issue again.

Understood, thanks for answering my question.

**Comment by Acclorite** [comment]
Will be implemented as per-folder access (with option to give access to multiple folders)

**Comment by Acclorite** [comment]
Preview of the feature: #179
